src/main/kotlin/com/vk/kphpstorm/exphptype/psi/TokensToExPhpTypePsiParsing.kt

```diff
@@ -183,6 +201,18 @@ internal object TokensToExPhpTypePsiParsing {
             return true
         }
 
+        if (builder.compare(PhpDocTokenTypes.DOC_IDENTIFIER) && (builder.tokenText == "ffi_cdata" || builder.tokenText == "ffi_scope")) {
+            val marker = builder.mark()
+            builder.advanceLexer()
+            if (!parseFfiContents(builder)) {
+                marker.drop()
+                return false
+            }
+
```

Comment:
(code style) No newline here, like in `if` above and below

Reply:
Fixed :)